aggregator/aggregator_mock.go

```diff
@@ -0,0 +1,467 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// Automatically generated by MockGen. DO NOT EDIT!
+// Source: github.com/m3db/m3aggregator/aggregator (interfaces: ElectionManager,FlushTimesManager,PeriodicFlusher,PlacementManager,Handler,Writer)
+
+package aggregator
+
+import (
+	"context"
+	"time"
+
+	"github.com/m3db/m3aggregator/generated/proto/flush"
+	"github.com/m3db/m3cluster/placement"
+	"github.com/m3db/m3cluster/shard"
+	"github.com/m3db/m3metrics/metric/aggregated"
+	"github.com/m3db/m3x/watch"
+	"github.com/uber-go/tally"
```

Comment:
looks like the `mockclean` utility is failing on the input file here (it should generate output with the correct import blocks). Mind opening an issue on `github.com/m3db/build-tools` to track

Reply:
Sure yeah can do

Reply:
Btw I put up a PR to fix this issue: https://github.com/m3db/ci-scripts/pull/35. It's not a mockclean issue per se. Rather, it's because the auto-generated files are not finding imports in the right location. In any case, feel free to take a look.